test/unit/concerns/background_deletion_test.rb

```diff
@@ -36,5 +36,20 @@ def test_delete_integration
       DeleteObjectHierarchyWorker.expects(:perform_later).with(user, anything, 'destroy').never
       DeleteObjectHierarchyWorker.perform_now(double_object)
     end
+
+    test 'all associations of the config can be constantized' do
+      Rails.application.eager_load!
+
+      models = ActiveRecord::Base.descendants - [DoubleObject, DoubleActiveRecordObject, SecondDoubleActiveRecordObject]
```

Comment:
```suggestion
      models = ActiveRecord::Base.descendants.select {|model| model.background_deletion.any? } 
```

Reply:
@hallelujah That wouldn't work because I am removing only the ones that are part of this test file, so it should be then:
models = `ActiveRecord::Base.descendants.select {|model| model.background_deletion.present? && ![DoubleObject, DoubleActiveRecordObject, SecondDoubleActiveRecordObject].include?(model) } `

However, I am not sure if it is worth to do a loop here for filtering and later another loop to treat it. Probably better just do in https://github.com/3scale/porta/pull/1685#discussion_r380162089
directly:
```ruby
ActiveRecord::Base.descendants.each do |klass|
  next if DoubleObject, DoubleActiveRecordObject, SecondDoubleActiveRecordObject].include?(model) || model.background_deletion.blank?
# ...
```